app/views/hyrax/collections/show.html.erb

```diff
@@ -1,33 +1,120 @@
-<% provide :page_title, construct_page_title(@presenter.title) %>
-<div itemscope itemtype="http://schema.org/CollectionPage" class="row collection">
-  <div class="col-sm-10 pull-right">
-    <header>
-      <%= render 'collection_title', presenter: @presenter %>
-    </header>
-    <%= render 'collection_description', presenter: @presenter %>
-
-    <% unless has_collection_search_parameters? %>
-      <%= render 'show_descriptions' %>
-    <% end %>
+<%
+   provide :page_title, construct_page_title(@presenter.title)
+
+   if banner_file(@presenter.id).blank?
```

Comment:
Please keep code out of the views. Consider using a presenter.

Reply:
Okay.

Reply:
✅
Developed a new solution for displaying this information as part of the "background image" update I made to the collections show code. As such, the code in question was no longer needed so I removed it.